### PR TITLE
Create InTalks.yml

### PR DESCRIPTION
**Pull request by josharsh**
Added InTalks as a project from https://github.com/josharsh/InTalks

### CODE REVIEW

File: _data/projects/InTalks.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: InTalks
+desc: InTalks is an android application used to fetch the latest news about a topic from various online news platforms.
+site: https://www.github.com/josharsh/Intalks
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Java
```
Comment (shiftkey) [inline _data/projects/InTalks.yml]:
```suggestion
- java
```

File: _data/projects/InTalks.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- Android
```
Comment (shiftkey) [inline _data/projects/InTalks.yml]:
```suggestion
- android
```

File: _data/projects/InTalks.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- API
```
Comment (shiftkey) [inline _data/projects/InTalks.yml]:
```suggestion
- api
```

File: _data/projects/InTalks.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+upforgrabs:
+  name: good first issue
+  link: https://github.com/josharsh/InTalks/blob/master/README.md
```
Comment (shiftkey) [inline _data/projects/InTalks.yml]: Can this link point to a list of curated issues? I don't see any created in the repository currently...

**Review by shiftkey — CHANGES_REQUESTED** [review]
@josharsh thanks for opening this up - there's some CI issues with the tags used that's easy to fix, and one question I have about the link used:

```
/home/runner/work/up-for-grabs.net/up-for-grabs.net/_data/projects/InTalks.yml - 
Tag verification failed!
Tag 'Java' contains uppercase characters
Tag 'Android' contains uppercase characters
Tag 'API' contains uppercase characters
##[error]Process completed with exit code 255.
```

**Comment by josharsh** [comment]
changes implemented

**Review by shiftkey — APPROVED** [review]
Thanks @josharsh!

**Pull request merged**